source/installation/resource-manager/kubernetes.rst

```diff
@@ -265,13 +319,87 @@ for more details.
 Kyverno Policies
 ****************
 
+Once Kubernetes is available to OnDemand, it's possible for users to use ``kubectl`` to submit arbirary pods to Kubernetes. To ensure proper security with Kubernetes a policy engine such as `Kyverno`_ can be used to ensure certain security standards.
+
+For OnDemand maybe of the `Kyverno baseline and restricted sescurity policies`_ will work.  There are also policies that can be deployed to ensure the UID/GID of user pods match that user's UID/GID on the HPC clusters.  Some `example policies`_ do things such as enforce UID/GID and other security standards for OnDemand. These policies rely heavily on the fact that OnDemand usage in Kubernetes using a namespace prefix.
+
+The policies enforcing UID/GID and supplemental groups are utilizing the `k8-ldap-configmap`_ tool that generates ConfigMap resources based on LDAP data.  This tool runs as a deployment inside the Kubernetes cluster.
+
 Enforcing Walltimes
 *******************
 
-TODO docs about the job-pod-reaper and k8-namespace-reaper
+In order to enforce that OnDemand pods have a finite runtime it's necessary to deploy a service that can cleanup pods that have run past their walltime.  Also because OnDemand is bootstrapping a namespace per user it's useful to cleanup unused namespaces.
```

Comment:
```suggestion
In order to enforce pods are shut down after so much time it's necessary to deploy a service that can cleanup pods that have run past their walltime.  Also because OnDemand is bootstrapping a namespace per user it's useful to cleanup unused namespaces.
```

Reply:
Made this change, couldn't use Github commit due to fact this is now outdated from line breaks I added a few minutes ago.